android/src/main/java/io/sentry/react/RNSentryModule.java

```diff
@@ -292,7 +293,16 @@ public void fetchNativeFrames(Promise promise) {
     }
 
     @ReactMethod
-    public void captureEnvelope(String envelope, Promise promise) {
+    public void captureEnvelope(ReadableArray rawBytes, ReadableMap options, Promise promise) {
+        if (!options.getBoolean("store")) {
+            logger.warning("Envelopes are always stored.");
+        }
```

Comment:
If this is intentional, we don't need to log it.

Reply:
okay, I'll remove it. I just wanted it to be clear that it doesn't have any effect on android behaviour.